docs/fr/documentation/schedule/reference.md

```diff
@@ -250,7 +250,7 @@
  | `route_id` | Clé étrangère faisant référence à `routes.route_id` | **Requis** | Identifie un itinéraire. | 
  | `service_id` | Clé étrangère faisant référence à `calendar.service_id` ou `calendar_dates.service_id` | **Requis** | Identifie un ensemble de dates auxquelles le service est disponible pour un ou plusieurs itinéraires. | 
  | `trip_id` | Identifiant unique | **Requis** | Identifie un voyage. | 
- | `trip_headsign` | Text | Optionnel | Text qui apparaît sur la signalisation identifiant la destination du voyage aux passagers. Doit être utilisé pour distinguer les différents modèles de service sur le même itinéraire.<br><br> Si le signe change pendant un voyage, les valeurs de `trip_headsign` peuvent être remplacées en définissant des valeurs dans `stop_times.stop_headsign` pour des `stop_time` spécifiques tout au long du voyage. | 
+ | `trip_headsign` | Text | Optionnel | Text qui apparaît sur la signalisation identifiant la destination du voyage aux passagers. Ce champ est recommandé pour tous les services avec une Girouette affichée sur le véhicule qui peut être utilisé pour distinguer les trajets d'un itinéraire.<br><br> Si le signe change pendant un voyage, les valeurs de `trip_headsign` peuvent être remplacées en définissant des valeurs dans `stop_times.stop_headsign` pour des `stop_time` spécifiques tout au long du voyage. | 
```

Comment:
Here we don't need "Girouette" to have a capital letter, it should be "girouette".

Reply:
I updated the text to reflect these corrections, including changing all mentions of "text" in the doc to "texte"